File: scaffold/src/main/java/top/xuqingquan/web/agent/JsAccessEntrace.java
package top.xuqingquan.web.agent;

public interface JsAccessEntrace extends QuickCallJs {

    void callJs(String js, android.webkit.ValueCallback<String> callback);

    void callJs(String js, com.tencent.smtt.sdk.ValueCallback<String> callback);

    void callJs(String js);

}

File: scaffold/src/main/java/top/xuqingquan/webbak/publics/JsAccessEntrace.kt
package top.xuqingquan.webbak.publics

interface JsAccessEntrace : QuickCallJs {

    fun callJs(js: String?, callback: android.webkit.ValueCallback<String>?)

    fun callJs(js: String?, callback: com.tencent.smtt.sdk.ValueCallback<String>?)

    fun callJs(js: String?)

}
